Tuist/Dependencies.swift

```diff
@@ -16,7 +16,7 @@ let dependencies = Dependencies(
             .package(url: "https://github.com/Flight-School/AnyCodable", from: "0.6.6"),
             .package(url: "https://github.com/rnine/Checksum.git", .upToNextMajor(from: "1.0.2")),
             .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMajor(from: "0.15.1")),
-            .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .exact("2.8.0")),
+            .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .exact("2.9.0")),
```

Comment:
Please update it in `Package.swift` as well 🙏

Reply:
Thanks 🙏